packages/boba/gateway/src/containers/airdrop/Airdrop.js

```diff
@@ -0,0 +1,373 @@
+              <br/>Please bridge {(snapValueL1 - l2BalanceOMG)*0.99} or more OMG to Boba to claim your airdrop.
+            </Typography>
+          }
+
+          {/* STATE 4 - INITIATED BUT TOO EARLY */}
+          {recordFoundL1 && (unlockL1time !== 0) && (isUnlocked === false) && 
+            <>
+            <Typography 
+              variant="body1" 
+              component="p" 
+              sx={{color: 'green', fontWeight: '700'}}
+            >
+              Airdrop initiated
+            </Typography>
+            <Typography 
+              variant="body2" 
+              component="p" 
+            >
+              The unlock time is {unlockL1time}. 
+              <br/>After that, you will be able to claim your L1 snapshot BOBA.
+            </Typography>
+            </>
+          }
+
+          {/* STATE 5 - INITIATED AND READY TO AIRDROP */}
+          {isUnlocked && 
+            <>
+              <Typography 
+                variant="body2" 
+                component="p" 
+              >
+                The unlock time of {unlockL1time} has passed. You can now claim your L1 snapshot BOBA.
+              </Typography>
+              <Button
+                onClick={(i)=>{this.airdropL1()}}
+                color="primary"
+                size="large"
+                newStyle
+                variant="contained"
+              >
+                Claim my L1 snapshot BOBA!
+              </Button>
+            </>
+          }
+           
+          {/* STATE 6 - CLAIMED */}
+          {!!claimedL1 &&
+            <>
+            <Typography 
+              variant="body1" 
+              component="p" 
+              sx={{color: 'green', fontWeight: '700'}}
+            >
+              Airdrop completed
+            </Typography>
+            <Typography 
+              variant="body2" 
+              component="p" 
+              sx={{mt: 1, mb: 2}}
+            >
+              You claimed your L1 snapshot BOBA at {claimedL1time}.
+            </Typography>
+            </>
+          }
+          </Box>
+        </Box>
+
+        <Box sx={{background: 'rgba(255, 255, 255, 0.07)', marginTop: '20px', padding: '10px', borderRadius: '10px'}}>
+          <Typography variant="h3" component="h3" sx={{fontWeight: "700",marginBottom: '20px'}}>L2 Airdrop</Typography>
+          <Box sx={{background: 'rgba(255, 255, 255, 0.11)', padding: '10px', borderRadius: '3px'}}>
+
+          {!recordFoundL2 &&
+              <Typography 
+              variant="body2" 
+              component="p" 
+            >
+              There is no record of OMG in this wallet on Boba during the snapshot.
+            </Typography>
+          }
+
+          {recordFoundL2 && snapValueL2 > 0 && claimedL2 === false &&
+            <>
+              <Typography 
+                variant="body2" 
+                component="p" 
+              >
+                There was a balance of {snapValueL2*0.95} OMG on Boba during the snapshot.
```

Comment:
should this be more accurate?
> {snapValueL2*0.95}

snapValueL2 * (100/105)

Reply:
Yes good eye! Fixing

Reply:
Let's go with that